models/us_census_cps/code/upload.py

```
@@ -0,0 +1,105 @@
+        )
+    return n
+
+
+def main():
+    only = set(sys.argv[1:])
+    tables = [(s, r) for s, r in TABLES if not only or s in only]
+    for slug, expected in tables:
+        print(f"=== {slug} ===", flush=True)
+        try:
+            upload_table(slug, expected)
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)